packages/cli/src/commands/verify.ts

```diff
@@ -0,0 +1,100 @@
+import type { CommandModule } from "yargs";
+import { verify } from "../verify";
+import { logError } from "../utils/errors";
+import { loadConfig } from "@latticexyz/config/node";
+import { World as WorldConfig } from "@latticexyz/world";
+import { resolveWorldConfig } from "@latticexyz/world/internal";
+import { worldToV1 } from "@latticexyz/world/config/v2";
+import { getOutDirectory, getSrcDirectory } from "@latticexyz/common/foundry";
+import { getExistingContracts } from "../utils/getExistingContracts";
+import { getContractData } from "../utils/getContractData";
+import { defaultModuleContracts } from "../utils/defaultModuleContracts";
+import { Hex } from "viem";
+import { MUDError } from "@latticexyz/common/errors";
+
+type Options = {
+  worldAddress?: string;
+  configPath?: string;
+  profile?: string;
+  srcDir?: string;
+  verifier?: string;
+  verifierUrl?: string;
+};
+
+const commandModule: CommandModule<Options, Options> = {
+  command: "verify",
+
+  describe: "Verify contracts",
+
+  builder(yargs) {
+    return yargs.options({
+      configPath: { type: "string", desc: "Path to the MUD config file" },
+      profile: { type: "string", desc: "The foundry profile to use" },
+      worldAddress: { type: "string", desc: "Verify an existing World at the given address" },
+      srcDir: { type: "string", desc: "Source directory. Defaults to foundry src directory." },
+      verifier: { type: "string", desc: "The verifier to use" },
+      verifierUrl: {
+        type: "string",
+        desc: "The verification provider.",
+      },
+    });
+  },
+
+  async handler(opts) {
+    if (!opts.worldAddress) {
+      throw new MUDError("Need a world address");
+    }
```

Comment:
this is not an optional arg right? so `yargs` would already throw if it's not provided

Reply:
Good shout, made it required!